packages/docs/pages/usage/search/filters.mdx

```diff
@@ -113,10 +113,20 @@ const results = await search(db, {
 
 ## Enum operators
 
-The numeric properties support the following operators:
+The enum properties support the following operators:
 
 | Operator  | Description                        | Example                                 |
 | --------- | ------------------------------     | ---------------------------------       |
 | `eq`      | Equal to                           | `genre: { eq: 'drama' }`                |
 | `in`      | Contained in the given array       | `genre: { in: ['drama', 'horror'] }`    |
 | `nin`     | Not contained in the given array   | `genre: { nin: ['commedy'] }`           |
+
+
+## Enum[] operators
+
+The enum properties support the following operators:
+
+| Operator        | Description                        | Example                                       |
+| ---------       | ------------------------------     | ---------------------------------             |
+| `contains`      | Contains the given value           | `genre: { contains: 'drama' }`                |
+| `intersects`    | Intersects with the given array    | `genre: { intersects: ['drama', 'horror'] }`  |
```

Comment:
Why use two different keywords? As far as I understand, `contains` is just a monomorphic version of `intersect`. We could also use `genre: { contains: ['drama', 'horror'] }` to avoid introducing an extra keyword.

Reply:
Because contains mean a subset, not an intersection:
- the field `enum[]` is an array
- I, as a developer, provide an `array`
How can I "compare" those arrays? 
- `eq` is wrong: it depends on what means equality in this case
- `contains` means "the doc field contains the given array.". Also, this is wrong. We are not calculating the subset, just the intersection.

We can collapse it into unique `contains` but it should be implemented differently (and heavier).

Reply:
What's the difference between:

`genre: { contains: 'drama' }`

and

`genre: { contains: ['drama'] }` ?